pftp/proxy.go

```diff
@@ -184,8 +179,8 @@ func (s *proxyServer) GetConn() net.Conn {
 func (s *proxyServer) SetDataHandler(handler *dataHandler) {
 	// only one data connection available in same time.
 	if s.dataConnector != nil {
-		// if already had previous data handler, wait until response sent to client.
-		for s.waitDataResponse {
+		// if already had previous data handler in use, wait until end.
+		for s.dataConnector.inUse {
```

Comment:
move status check variable into the datahandler structure.
because In proxy structure, it is too hard to update when transfer job completed.

Reply:
I think that You had better to use channel because chanel is useful in this case.

Reply:
https://github.com/pyama86/pftp/pull/35/commits/7a0aeb43aaad21d534ffb198887b3af4cf929d0a

Fixed!